### PR TITLE
Manual merge external to internal

### PR DESCRIPTION
**Pull request by oscarsernarosero**
There are a couple of changes I would like to have special attention:

- Since we are most likely work on some braking changes, I disabled checkApplicationRules in the AMM handler.
- I removed action from the AMM handler.
- I moved all tests for Purchase Limit, Purchase Percentage, Sell Limit, and Sell Percentage out of the AMM since now they are in the tokens.

### CODE REVIEW

File: docs/userGuides/rules/ACCOUNT-PURCHASE-RULE.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ The Account Purchase Rule is an account based measure which restricts an account
 
 ## Applies To:
 
+<<<<<<< HEAD
```
Comment (TJ-Everett) [inline docs/userGuides/rules/ACCOUNT-PURCHASE-RULE.md]: I think we missed some merge syntax here

File: docs/userGuides/rules/ACCOUNT-PURCHASE-RULE.md [hunk 2]
```diff
@@ -21,6 +22,22 @@ As this is a [tag](../GLOSSARY.md)-based rule, you can think of it as a collecti
 - **Purchase Amounts** (uint192): The maximum amount of tokens that may be purchased during the *purchase period*. 
 - **Purchase Periods** (uint16): The length of each time period for which the rule will apply, in hours.
 - **Starting Timestamp** (uint64): The Unix timestamp of the date when the *period* starts counting.
+=======
```
Comment (TJ-Everett) [inline docs/userGuides/rules/ACCOUNT-PURCHASE-RULE.md]: and here

File: docs/userGuides/rules/TOKEN-PERCENTAGE-SELL.md [hunk 3]
```diff
@@ -156,7 +186,11 @@ The following validation will be carried out by the create function in order to
             view 
             returns (uint256);
         ```
+<<<<<<< HEAD
```
Comment (ShaneDuncan602) [inline docs/userGuides/rules/TOKEN-PERCENTAGE-SELL.md]: There are a bunch of these issues in the .md files. It looks like you may have skipped them.

**Review by ShaneDuncan602 — CHANGES_REQUESTED** [review]
There are still a bunch of merge conflicts. sear for <<<<< and you will see them.

**Review by ShaneDuncan602 — APPROVED** [review]
Looks good.

**Review by TJ-Everett — APPROVED** [review]
Looks good Oscar

**Review by mpetersoCode55 — APPROVED** [review]

**Pull request merged**